README.md

```diff
@@ -17,13 +17,13 @@ Current version of the library requires a latest stable Go release. If you don't
 Use go tool to install the package in your packages tree:
 
 ```
-go get github.com/trustmaster/goflow
+go get github.com/Synthace/goflow
```

Comment:
The URL should be changed back to the original repo. Make another branch for the PR if you want to keep it this way for Synthace.